src/connectors/toggle-refinement/connectToggleRefinement.js

```diff
@@ -306,6 +307,7 @@ export default function connectToggleRefinement(renderFn, unmountFn = noop) {
             createURL,
           }),
           sendEvent,
+          canRefine: Boolean(results ? nextRefinement.count : null),
```

Comment:
I'm not sure what condition it should be here. Maybe to be false, but current refinement and next refinement should have a 0 count? I'm not sure!

Reply:
This condition is actually coming from Vue InstantSearch

```js
const mapStateToCanRefine = state => Boolean(state.value && state.value.count);
```

and `state.value` is made in InstantSearch.js like this:

```js
          value: {
            name: attribute,
            isRefined,
            count: results ? nextRefinement.count : null,
            onFacetValue,
            offFacetValue,
          },
```

I'm not too sure either, but it'd be good to align between InstantSearch.js and Vue InstantSearch first, and then later change to something else if needed from the InstantSearch.js and propagate to Vue InstantSearch. What do you think?